samples/api-import/import-sample.xsjs

```diff
@@ -1,7 +1,7 @@
 // Import .xsjslib to our file using $.import api ("package","library")
 // In our case package equals to "sap.myapp.lib"
 
-let mathlib = $.import("demo.lib","math");
+let mathlib = $.import("/api-import/demo.lib","math");
```

Comment:
Shouldn't the import path be `api-import.demo.lib`?

Reply:
Yep. works with dots as well. 

Will change it as the dot syntax is convention.